tox.ini

```diff
@@ -0,0 +1,6 @@
+[tox]
```

Comment:
this is pre-emptive as tox depends on a setup.py, but it is really handy if we want to support multiple versions of python

https://testrun.org/tox/latest/example/pytest.html